### ISSUE TITLE
cc license image missing alt text / should be hidden

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#697]
Example pages
https://earthworks.stanford.edu/catalog/stanford-nr366rs0962
https://earthworks.stanford.edu/catalog/stanford-vc063fm2579

The cc images are reported for not having alt text - but since the text equivalent immediately follows the image, the image itself should just be aria-hidden to avoid redundancy. 

- [ ] add `aria-hidden="true"` to the `<img>` tag
